jibri/rootfs/etc/services.d/10-xorg/run

```diff
@@ -1,5 +1,5 @@
-#!/usr/bin/with-contenv bash
+#!/command/with-contenv bash
 
 DAEMON="/usr/bin/Xorg -nocursor -noreset  +extension RANDR +extension RENDER -logfile /tmp/xorg.log  -config /etc/jitsi/jibri/xorg-video-dummy.conf ${DISPLAY}"
-exec s6-setuidgid jibri /bin/bash -c "exec $DAEMON"
+exec s6-setuidgid /bin/bash -c "exec $DAEMON"
```

Comment:
Why do we still need s6-setuidgid if we are not specifying the user?

Reply:
deleted